app/models/manageiq/providers/redhat/infra_manager/event_target_parser.rb

```diff
@@ -0,0 +1,90 @@
+class ManageIQ::Providers::Redhat::InfraManager::EventTargetParser
+  attr_reader :ems_event
+
+  # @param ems_event [EmsEvent] EmsEvent object
+  def initialize(ems_event)
+    @ems_event = ems_event
+  end
+
+  # Parses all targets that are present in the EmsEvent given in the initializer
+  #
+  # @return [Array] Array of InventoryRefresh::Target objects
+  def parse
+    parse_ems_event_targets(ems_event)
+  end
+
+  private
+
+  # Parses list of InventoryRefresh::Target out of the given EmsEvent
+  #
+  # @param event [EmsEvent] EmsEvent object
+  # @return [Array] Array of InventoryRefresh::Target objects
+  def parse_ems_event_targets(event)
+    target_collection = InventoryRefresh::TargetCollection.new(:manager => event.ext_management_system, :event => event)
+
+    # there's almost always a tenant id regardless of event type
+    collect_identity_tenant_references!(target_collection, ems_event)
+
+    if ems_event.event_type.start_with?("compute.instance")
```

Comment:
Does RHV has events that start with `"compute.instance"` ?  I don't see any of these events in https://github.com/ManageIQ/manageiq-content/tree/master/content/automate/ManageIQ/System/Event/EmsEvent/RHEVM.class

It looks like they were copied from OpenStack

Reply:
oh they were

Reply:
Okay lets remove this set of checks as well as https://github.com/ManageIQ/manageiq-providers-ovirt/pull/487/files#diff-26c9135e0247d314dab264c48e4a728bR26 and all these https://github.com/ManageIQ/manageiq-providers-ovirt/pull/487/files#diff-26c9135e0247d314dab264c48e4a728bR51-R89